Jenkinsfile

```diff
@@ -522,8 +522,8 @@ pipeline {
             Build for aarch64_mac platform')
         booleanParam(name: 'x86_64_mac', defaultValue: true, description: '\
             Build for x86-64_mac platform')
-        booleanParam(name: 'aarch64_linux', defaultValue: true, description: '\
-            Build for aarch64_linux platform')
+        booleanParam(name: 'AARCH64_LINUX', defaultValue: true, description: '\
```

Comment:
Any particular reason why this is in capital letters?

Reply:
Sorry this was a mistake and was reverted.